### PR TITLE
build: Update plan stability for Spark 4.0

### PR DESCRIPTION
**Pull request by viirya**
## Which issue does this PR close?

Closes #.

## Rationale for this change

Fixing broken CI pipelines.

### CODE REVIEW

**Comment by viirya** [comment]
The plan stability results of Spark 4.0 is changed. It causes current CI pipelines failures.

**Comment by viirya** [comment]
cc @kazuyukitanimura @andygrove @huaxingao

File: spark/src/test/resources/tpcds-plan-stability/approved-plans-v1_4-spark4_0/q1/explain.txt [hunk 1]
```diff
@@ -31,9 +31,9 @@ TakeOrderedAndProject (40)
       :     :                                   :     +- CometScan parquet spark_catalog.default.store_returns (11)
       :     :                                   +- ReusedExchange (14)
       :     +- BroadcastExchange (31)
-      :        +- * ColumnarToRow (30)
-      :           +- CometProject (29)
-      :              +- CometFilter (28)
+      :        +- * Project (30)
+      :           +- * Filter (29)
+      :              +- * ColumnarToRow (28)
```
Comment (kazuyukitanimura) [inline spark/src/test/resources/tpcds-plan-stability/approved-plans-v1_4-spark4_0/q1/explain.txt:36]: Hmmm do you know why CometFilter is no longer activated by any chance?

Comment (viirya) [reply spark/src/test/resources/tpcds-plan-stability/approved-plans-v1_4-spark4_0/q1/explain.txt:36]: No. But just want to restore CI as soon as possible. We can investigate it later.

**Comment by viirya** [comment]
I guess that with #526, we don't need to update the plan stability.

**Pull request closed without merging**